cmd/influx/cli/cli.go

```diff
@@ -31,6 +31,10 @@ import (
 // ErrBlankCommand is returned when a parsed command is empty.
 var ErrBlankCommand = errors.New("empty input")
 
+// DefaultChunkSize is the default number of results to return
+// per response if chunked responses are turned on
+const DefaultChunkSize = 10000
```

Comment:
ah, that makes sense.  I see that the handler already sets to the default if we pass zero.